README.Rmd

```diff
@@ -20,6 +20,8 @@ knitr::opts_chunk$set(
 
 The R Package `stagg` aims to harmonize the preparation of spatiotemporal data obtained from the [ERA5 project](https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era5) for use in statistical analyses. This allows for more efficient useof researchers' time, and avoids common missteps. The end goal is a greater quantity of quality research into the study of coupled human-geophysical systems. 
 
+Further documentation, including the working paper, AGU conference poster, and cheat sheet are available [here](https://www.tammacarleton.com/)
```

Comment:
@tliddell4 please change URL to https://www.tammacarleton.com/projects-6

Reply:
@tcarleton i changed the URL and added to the gitignore.